debugging-3/features/episodes.feature

```diff
@@ -0,0 +1,13 @@
+Feature: Episodes
+
+  @100
+  Scenario: TV Shows are displayed
+    Given I am an end user
+    When I enter "bob" as an email
```

Comment:
Is this a typo? (email scenario in episodes test) And same for below

Reply:
Copy past error. Have fixed it